src/gui/guiBox.cpp

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Needs copyright header

Reply:
Should I just copy it from another file?

Reply:
Yes, or directly from the source: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt

Reply:
If so, done.

Reply:
Is it correct how it is currently?